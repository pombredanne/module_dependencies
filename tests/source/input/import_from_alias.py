from nltk.corpus import brown as br
from nltk.tokenize import TextTilingTokenizer as ttt

output = ttt().tokenize(br.raw()[0:10000])
