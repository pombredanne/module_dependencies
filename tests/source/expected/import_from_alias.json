{
    "dependencies": [
        "nltk.corpus.brown.raw",
        "nltk.tokenize.TextTilingTokenizer"
    ],
    "imports": [
        "nltk.corpus",
        "nltk.tokenize"
    ]
}
